src/app/routes/article/getInitialData/index.ts

```diff
@@ -19,6 +20,16 @@ type Props = {
   path: string;
   pageType: 'article' | 'cpsAsset';
   variant?: Variants;
+  toggles?: Toggles;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const addDisclaimer = async (pageData: any, toggles: Toggles | undefined) => {
```

Comment:
Used a comment to stop this giving me a warning. Not sure if there is a type I can use here?

Reply:
There isn't... yet. I do wonder if at some point we need to bite the bullet and add a global pageData definition. (not in this PR)